### DISCUSSION TITLE
Paypal does not support this bock (storefront theme) in checkout and basket

### DISCUSSION THREAD

**Discussion by sandyRD** [woocommerce/woocommerce#45918] [Checkout Flow]
i recently took this theme to make a store web, but cant make checkout and basket work with paypal and cards extensions or plugin, it says at both pages : Paypal does not support this bock (storefront theme) in checkout and basket , i have erase both and reinstall new automatic last version woocommerce version, but not working, any suggestion?

**Comment by ralucaStan** [comment]
@sandyRD could you provide some replication steps and some screenshots? Those would help with understanding your issue better
